### PR TITLE
Autopublish releases to npm

### PR DESCRIPTION
**Pull request by FlattestWhite**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@setprotocol/index-coop-contracts",
+  "name": "@indexcoop/index-coop-contracts",
```
Comment (0xModene) [inline package.json:2]: should be 
```typescript
"name": "@indexcoop/index-coop-smart-contracts",
```

**Review by 0xModene — CHANGES_REQUESTED** [review]
small change needed but mostly looks good

File: package.json (hunk 1 above)
Comment (FlattestWhite) [reply package.json]: whoops - fixed

**Comment by FlattestWhite** [comment]
Just a note on this PR - circle ci atm isn't actually verifying this repo because the yarn.lock wasn't regenerated when porting it over. We cache based on the yarn.lock file. Since the yarn.lock file remains the same, the old build and test steps still pass but its pointing to ~/index-coop-private

**Comment by 0xModene** [comment]
@FlattestWhite Can you run `yarn` locally and resolve this?

**Comment by FlattestWhite** [comment]
> @FlattestWhite Can you run `yarn` locally and resolve this?

done

**Review by 0xModene — APPROVED** [review]

**Pull request merged**